easybuild/easyconfigs/l/libleidenalg/libleidenalg-0.11.1-foss-2022b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'libleidenalg'
+version = '0.11.1'
+
+homepage = 'https://github.com/vtraag/libleidenalg'
+description = """Implements the Leiden algorithm in C++"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+source_urls = ['https://github.com/vtraag/libleidenalg/archive/refs/tags']
+sources = ['%(version)s.tar.gz']
+checksums = ['7d7392afd214c584e023cc2f0d0ac375f58575c32f2e49ba85062065f1637c7f']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('binutils', '2.39'),
+]
+
+dependencies = [
+    ('igraph', '0.10.6'),
+]
+
+sanity_check_paths = {
+    'files': ['include/%(name)s/GraphHelper.h', 'include/%(name)s/libleidenalg_export.h', 'lib/lib%(name)s.so'],
```

Comment:
Please use SHLIB_EXT (see https://docs.easybuild.io/version-specific/easyconfig-templates/#template-constants-that-can-be-used-in-easyconfigs) instead of hardcodeing the `.so` extension.

Reply:
ok